data/mapcss-mapping.csv

```diff
@@ -1336,7 +1336,7 @@ railway|subway_entrance|warszawa;[railway=subway_entrance][city=warszawa];;name;
 shop|bed;1221;
 shop|outpost;1222;
 shop|gas;1223;
-natural|desert;[natural=desert],[natural=sand][desert=erg];;name;int_name;1224;
+natural|sand;1224;
```

Comment:
@pastk Do you know what the meaning of `name;int_name` is? Does removing it change anything or is it only important when `int_name` is used in the styles (like e.g. for neighbourhood)?

Reply:
they're not used at all, its safe to drop them, e.g. `natural|rock;[natural=rock],[natural=stone];;;;969;`